lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
+  description: Payment method codes
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
+  binding:
+    description: Payment method codes
```

Comment:
The payment method of the transaction.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/5b9fb3f4a9e5043e58d9e314658b92f5e3b3e16f#diff-5125c86cde87f8439cb9d1bb859194e9R9

Reply:
+1